core-services/ci-secret-bootstrap/_config.yaml

```diff
@@ -252,6 +252,9 @@
     - cluster: api.ci
       namespace: ci
       name: github-credentials-openshift-ci-robot-private-git-cloner
+    - cluster: app.ci
+      namespace: ci
+      name: github-credentials-openshift-ci-robot-private-git-cloner
```

Comment:
Why do we need this exactly? Nothing that clones repo should run on app.ci.

Reply:
`group-auto-updater` job is updating the necessary openshift private group that manages the permissions in `deck-internal`

Reply:
Argh, I see. We are abusing this token to talk to GH so that we can read org membership :/